website/src/components/Header.tsx

```diff
@@ -174,6 +183,9 @@ export default class Header extends React.Component<HeaderProps, any> {
                                                 <li className="nav-link" role="presentation">
                                                     <a href="/batch-changes">Batch Changes</a>
                                                 </li>
+                                                <li className="nav-link" role="presentation">
+                                                    <a href="/code-insights">Code Insights</a>
```

Comment:
Agreed! However, it is a larger cleanup job to deal with all of these right now, pre-launch, and I would rather do them all at once to achieve consistency across the site. I have created a [cleanup issue](https://github.com/sourcegraph/about/issues/5164), which I will address post-launch.

Reply:
I think, given the circumstances, we can now roll this in. Do you have any objections to rolling that issue into this PR?

Reply:
Thinking about this more, I think the cleanup should be a separate PR. I will work on that and we can merge it before we merge this one.

Reply:
Yeah, I mean, we could do an incremental approach and just do the Link elements in this PR since there are only a few. Then at least this page is improved. But for a larger scoped ticket like #5164, since we're re-platforming, those will be replaced with NextJS Link elements anyway. So the question is; do we even bother doing the entire site right now?

Reply:
That is a good question! What if we split the difference and scope #5164 to be "Use Link elements for internal links on nav and homepage"? Then we at least make improvements to the entirety of the page/ component we are touching in this PR

Reply:
That sounds good to me! Let's do that and split the difference.

Reply:
And then I will update these here